numpy/lib/type_check.py

```diff
@@ -271,6 +275,25 @@ def isreal(x):
     --------
     >>> np.isreal([1+1j, 1+0j, 4.5, 3, 2, 2j])
```

Comment:
@rossbar i wonder if i should leave this as is or create an array and apply the function on the array for consistency. if latter, something like `a = np.isreal([1+1j, 1+0j, 4.5, 3, 2, 2j], dtype='c16')` and then apply `np.isreal(a)`

Reply:
I agree that being explicit about dtypes when creating arrays is generally a good practice; though this case is straightforward. I'd say it's up to you :) - though I think you meant `np.array` instead of `np.isreal` in your suggestion, and I would use the more generic `dtype=complex`.

Reply:
yep, thanks!